index.html

```diff
@@ -204,6 +204,7 @@ <h2 id="footer-heading">CONTACT US</h2>
      </form>
```

Comment:
- [ ] please make sure to implement the effect button on hover and on pressed same as the Figma design as the previous reviewer asked